vpc/service/db/migrations/38_trunk_eni_id.down.sql

```diff
@@ -0,0 +1,7 @@
+START TRANSACTION;
+LOCK TABLE htb_classid IN ACCESS EXCLUSIVE MODE;
+LOCK TABLE trunk_enis IN ACCESS EXCLUSIVE MODE;
+ALTER SEQUENCE trunk_enis_id_seq AS INTEGER NO CYCLE;
```

Comment:
Does it matter that this is back to `int` even though during the incident resolution it was marked as `bigint`?

Reply:
it'll be a problem if we've exceeeded 2**31.

Reply:
@MyUmmaGumma thats the `down` migration file to migrate down to a previous version.

Reply:
Understood, I was wondering if the previous version is one on the schema or the one on the box(which is bigint now)